pkg/bmc/access_test.go

```diff
@@ -879,6 +921,52 @@ func TestDriverInfo(t *testing.T) {
 				"ibmc_verify_ca": false,
 			},
 		},
+
+		{
+			Scenario: "ilo4",
+			input:    "ilo4://192.168.122.1",
+			expects: map[string]interface{}{
+				"ilo_address":   "192.168.122.1",
+				"ilo_password":  "",
+				"ilo_username":  "",
+				"ilo_verify_ca": false,
+			},
+		},
+
+		{
+                        Scenario: "ilo4 port",
+			input:    "ilo4://192.168.122.1:8080",
+                        expects: map[string]interface{}{
+                                "ilo_address":   "192.168.122.1",
+				"client_port":   "8080",
```

Comment:
The indentation looks off here. Could you make sure you run `make fmt` on the patch?